draft-regext-rdap-extensions.md

```diff
@@ -193,4 +234,36 @@ with the use of JSON values from other extensions or RDAP specifications. As wit
 the ICANN profile or RIR profile extensions, the extension may simply signal 
 policy applied to already defined RDAP structures.
 
+# Existing Extension Registrations
+
+The following extensions have been registered with IANA, but do not
+comply with the requirements set out in the base specifications, as
+clarified by this document:
```

Comment:
To be fair to these non-compliant extensions, is it worth elaborating a bit how they are non-compliant? That may become a "what not to do" lesson.

Reply:
I think it's implicit from the text following this part, or at least I think elaborating their non-compliance is effectively restating what's in that text.  It might be that an appendix reciting the history is a better place for going into detail about how people may have got confused about identifiers vs. conformance codes vs. JSON/URL prefixes, but if you have suggested text that could be included here please let me know.

Reply:
Agreed, we don't inadvertently want to amplify that non-compliance here. :) Yah, perhaps when we get to history. Though I wonder if such a history could be told, or is that worthwhile to do?

Reply:
I think it could be documented to some extent, roughly along the lines of some of the mail I sent to the list when this was discussed last year, but yep, it's kind of a toss up as to whether it's worth it.